java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java

```diff
@@ -147,19 +148,19 @@ public static MerkleTreeLeaf parseMerkleTreeLeaf(InputStream in) {
   }
 
   /**
-   * Parses a {@link Ct.TimestampedEntry} from binary encoding.
+   * Parses a {@link TimestampedEntry} from binary encoding.
    * @param in byte stream of binary encoding.
-   * @return Built {@link Ct.TimestampedEntry}.
+   * @return Built {@link TimestampedEntry}.
    * @throws SerializationException if the data stream is too short.
    */
-  public static Ct.TimestampedEntry parseTimestampedEntry(InputStream in) {
-    Ct.TimestampedEntry.Builder timestampedEntry = Ct.TimestampedEntry.newBuilder();
+  public static TimestampedEntry parseTimestampedEntry(InputStream in) {
+    TimestampedEntry timestampedEntry = new TimestampedEntry();
 
     long timestamp = readNumber(in, CTConstants.TIMESTAMP_LENGTH);
-    timestampedEntry.setTimestamp(timestamp);
+    timestampedEntry.timestamp = timestamp;
```

Comment:
I know you're doing a straight conversion of the old code, but these pairs of lines look like they ought to just be folded into one;  eg. timestampedEntry.timestamp = readNumber(in, CTConstants.TIMESTAMP_LENGTH);